### PR TITLE
Support other languages

### PR DESCRIPTION
**Pull request by ledong0110**
- Format data standard
- Metric configuration
- Language specific configuration

**Progress**:
- Complete 9 scenarios except IR task

### CODE REVIEW

**Comment by ledong0110** [comment]
Please help me verify the pipeline and wait for updating IR task

File: config/vi/llm_template.json
Comment (martinakaduc) [inline config/vi/llm_template.json:1]: LLM Template should be at root of `config` folder.

File: config/vi/summac_model.json
Comment (martinakaduc) [inline config/vi/summac_model.json:1]: I think this file should be in the root of `config` folder.

**Review by martinakaduc — CHANGES_REQUESTED** [review]
I think we need to update readme.

**Comment by sangttruong** [comment]
Too many files have changed -- could you provide a tutorial or a readme for others to understand the code structure?

**Comment by sangttruong** [comment]
@ledong0110 please resolve the conflict

**Pull request merged**